sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -16811,9 +16812,9 @@
               "tags": [],
               "properties": {},
               "conditional": {
-                "show": null,
-                "when": null,
-                "eq": "",
+                "show": "true",
```

Comment:
Why do we have these change? I see the ticket is changing only the supporting or partner forms. Am i missing something?

Reply:
okay I am good, just read the AC